src/metabase/driver/duckdb.clj

```diff
@@ -15,71 +15,68 @@
             ResultSet
             ResultSetMetaData
             Statement
-            Types]))
+            Time
+            Types]
+           [java.time LocalDate LocalTime OffsetTime]
+           [java.time.temporal ChronoField]
+           ))
 
 (driver/register! :duckdb, :parent :sql-jdbc)
 
 (defn- jdbc-spec
   "Creates a spec for `clojure.java.jdbc` to use for connecting to DuckDB via JDBC from the given `opts`"
-  [{:keys [database_file, read_only, allow_unsigned_extensions, old_implicit_casting, motherduck_token], :as details}]
```

Comment:
just removing the `let` expression here since `result` is only used once (as the return value)